DrawerKit/DrawerKit/Public API/DrawerDisplayController.swift

```diff
@@ -15,6 +15,9 @@ public final class DrawerDisplayController: NSObject {
     weak var presentingVC: UIViewController?
     /* strong */ var presentedVC: (UIViewController & DrawerPresentable)
 
+    let presentingDrawerAnimationActions: DrawerAnimationActions
+    let presentedDrawerAnimationActions: DrawerAnimationActions
```

Comment:
why not private?

Reply:
@RuiAAPeres They need to be internal because they're accessed outside of this compilation unit.

Reply:
where?

Reply:
`DrawerDisplayController+Extensions.swift` which injects them into `PresentationController` and `AnimationController.swift`, and they get used in `PresentationController+Animation.swift` and `AnimationSupport.swift`.